blocks/simple-list-block/features/simple-list/default.jsx

```diff
@@ -162,13 +152,13 @@ const SimpleList = (props) => {
 									placeholderResizedImageOptions={placeholderResizedImageOptions}
 									targetFallbackImage={targetFallbackImage}
 									arcSite={arcSite}
-									imageProps={imageProps}
+									primaryLogoAlt={primaryLogoAlt}
 								/>
 								<hr />
```

Comment:
This hr can be removed. Instead, the `divider` prop can be used in the stack 

https://61eeeb27215b5f003acdf18f-ojyukizixo.chromatic.com/?path=/story/components-stack--vertical-with-divider

`<Stack divider>`

Reply:
Added!

Reply:
nice nice